pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -247,8 +249,8 @@ func getLatestConfig(url string, iteration int,
 		return false
 	}
 
-	if err := validateProtoMessage(url, resp); err != nil {
-		log.Errorln("validateProtoMessage: ", err)
+	if err := utils.ValidateProtoMessage(url, resp); err != nil {
```

Comment:
This doesn't belong in a bugfix and it doesn't seem useful to just move some code around. Please drop from PR.

Reply:
I will update the PR.